src/domainobjectfactories/depot_position_factory.py

```diff
@@ -27,57 +27,106 @@ def create(self, record_count, start_id):
             Containing 'record_count' depot positions
         """
 
-        self.instruments = self.retrieve_records('instruments')
-
         records = []
 
         for _ in range(start_id, start_id+record_count):
-            instrument = self.get_random_instrument()
-            records.append(self.create_record(instrument))
+            records.append(self.__create_record())
 
         return records
 
-    def create_record(self, instrument):
+    def __create_record(self):
         """ Create a single depot position
 
-        Parameters
-        ----------
-        instrument : list
-            Dictionary containing a partial record of an instrument, only
-            containing information necessary to create depot positions.
-
         Returns
         -------
         dict
             A single depot position object
         """
 
-        position_type = self.create_position_type()
-        knowledge_date = self.create_knowledge_date()
+        isin, cusip, market = self.__create_instrument_details()
```

Comment:
I know we've been a bit inconsistent with naming, but using "create" rather than "get" here is definitely misleading